recipes/aqtest/meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: aqtest
+  version: "0.0.5"
+
+build:
+  noarch: python
+  number: 0
+
+source:
+  url: https://github.com/LYC-vio/Aqtest/archive/v0.0.5.tar.gz
+  sha256: 3c269aad7d9180faaca306d1b667965e3f15a208b9de46ec4e2e0dfca5f0204b
+
+requirements:
+  build:
+    - python
+    - setuptools >=1.1
+    - samtools
```

Comment:
The only things you actually need for installation are likely python and setuptools, the rest can be relegated to `run:`.

Reply:
thanks a bunch! :) I'm kind of new to this business, these suggestions are really helpful.